src/linux_mcp_server/commands.py

```diff
@@ -0,0 +1,196 @@
+"""Command registry for unified local and remote execution.
+
+This module provides a centralized registry of commands used by tools,
+enabling consistent execution across local and remote systems.
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
Since this project uses Pydantic, use that in stead of `dataclasses`.

Reply:
Oh I just changed the data types in `types.py`, not the command specs. 🤦‍♂️